dbms/src/Flash/Coprocessor/ExecutionSummary.cpp

```diff
@@ -34,6 +35,14 @@ void ExecutionSummary::merge(const tipb::ExecutorExecutionSummary & other)
     scan_context->merge(other.tiflash_scan_context());
 }
 
+void ExecutionSummary::set(const BaseRuntimeStatistics & other)
+{
+    time_processed_ns = other.execution_time_ns;
+    num_produced_rows += other.rows;
+    num_iterations += other.blocks;
+    concurrency += other.concurrency;
+}
```

Comment:
It looks like `ExecutionSummary::set` will only be called once, maybe should be 
```
void ExecutionSummary::fill(const BaseRuntimeStatistics & other)
{
    time_processed_ns = other.execution_time_ns;
    num_produced_rows = other.rows;
    num_iterations = other.blocks;
    concurrency = other.concurrency;
}
```
?

Reply:
> It looks like `ExecutionSummary::set` will only be called once, maybe should be

Good point